### PR TITLE
added-coins-table

### PR DESCRIPTION
**Pull request by jvin08**

### CODE REVIEW

File: app/components/coinsTable/Chart.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                gradient.addColorStop(0.15, "rgba(120, 120, 250, 0)");
+                return gradient;
+            },
+            },
+        ],
+        };
+    return (
+      <div className="ml-8">
+        {isLoaded ? <Line options={options} data={chartData} width={100} height={60} /> : <div className="loading"></div>}
+        {/* <div className="flex w-28 justify-center"><div className="loading"></div></div> */}
```
Comment (yoloOnTheBattlefield) [inline app/components/coinsTable/Chart.tsx]: delete this

File: app/components/coinsTable/CoinsTable.tsx [hunk 2]
```diff
@@ -4,31 +4,33 @@ import InfiniteScroll from "react-infinite-scroll-component";
...
+      <Header />
         <InfiniteScroll
           dataLength={coins.length}
           next={fetchMoreData}
           hasMore={true}
           loader={<h4>Loading...</h4>}
         >
           {coins?.map((coin: any, index: number) => (
-                <CoinCard key={index} coin={index+1} />
+                <CoinCard key={uid()} coin={coin} index={index} />
```
Comment (yoloOnTheBattlefield) [inline app/components/coinsTable/CoinsTable.tsx]: don;t use uid as key

File: app/components/coinsTable/PricePercentage.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import clsx from "clsx";
+const PricePercentage = ({price} : {price: number}) => {
+    const isPositive = (number:number) => {
+        return number > 0 ? true : false;
+    };
```
Comment (yoloOnTheBattlefield) [inline app/components/coinsTable/PricePercentage.tsx]: remove indent

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: app/components/coinsTable/Chart.tsx (hunk 1 above)
Comment (jvin08) [reply app/components/coinsTable/Chart.tsx]: fixed

File: app/components/coinsTable/CoinsTable.tsx (hunk 2 above)
Comment (jvin08) [reply app/components/coinsTable/CoinsTable.tsx]: uid replaced

File: app/components/coinsTable/PricePercentage.tsx (hunk 3 above)
Comment (jvin08) [reply app/components/coinsTable/PricePercentage.tsx]: indent removed

**Pull request merged**